README.md

```diff
@@ -0,0 +1,56 @@
+## Dispatch Event Driver for AWS
+This repo contains a template for aws SQS event driver support for Dispatch.
+
+### 1. Get AWS credentials
+Follow [`Get AWS Access Key`](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/setting-up.html) to obtain aws credential. Modify `<ACCESS_KEY_ID>` and `<SECRET_ACCESS_KEY>` in **credentials** file under this repo:
```

Comment:
baking credentials into image does not sound like a good idea. Could you make them configurable at the runtime? Basically accept those as environment variables or command line flags (you can pass them when creating an instance of event driver via `--config` or `--secret`). Vcenter driver already does that, you can take a look at it.

Reply:
fixed, passing credentials when create eventdriver.